app/access/tests/test_api.py

```diff
@@ -0,0 +1,110 @@
+from access.api import router
+from access.api import user_to_response
+from access.models import User
+from access.schemas import UserSchema
+from ninja.testing import TestClient
+from provider.models import Provider
+
+from django.test import TestCase
+
+
+class ApiTestCase(TestCase):
+
+    def setUp(self):
```

Comment:
I don't know if my setup is wrong here but mypy says that the functions lack a return type definition

Reply:
Tests should be excluded: https://github.com/geoadmin/service-control/blob/feat-PB-1045-add-user-model-and-endpoint/mypy.ini#L7. Maybe reload vscode?

Reply:
Hmm nope... do you guys also use ms-python.mypy-type-checker in vscode?

Reply:
I haven't used the [Mypy Type Checker extension](https://github.com/microsoft/vscode-mypy) yet but good idea.

I had to adjust some settings such that it takes our mypy with the corresponding settings:
```json
"mypy-type-checker.args": [
    "[\"--config-file=mypy.ini\"]"
],
"mypy-type-checker.importStrategy": "fromEnvironment"
```
Would it help to mention this in the README or is this too IDE specific?

Reply:
I would actually rather add this to [geoadmin/ansible-bgdi-dev](https://github.com/geoadmin/infra-ansible-bgdi-dev/pull/165)

Reply:
Nice, I see that you added it already [in this PR to infra-ansible-bgdi-dev](https://github.com/geoadmin/infra-ansible-bgdi-dev/pull/165). I resolve this thread, then.